content_dev/docs/sdk/java/_index.md

```diff
@@ -3,11 +3,10 @@ layout: list
 
 title: "Java Reference" # The title (ON THE PAGE)
 lead: | # The lead below the title (ON THE PAGE)
-  Build an atPlatform application in Java.
+  Build a Java application using the atPlatform.
 
-description:
-  | # SEO Description of the page (Shows in google and atsign.dev search)
-  My SEO description
+description: | # SEO Description of the page (Shows in google and atsign.dev search)
+    Implement security to your Java applications or IoT devices running Java
```

Comment:
We're not "implementing security". How about something like, "Add end-to-end encryption and data privacy to your Java applications and IoT devices running Java"

Reply:
5cb13e049ea795ebccea81420dcca070658f4edb